### DISCUSSION TITLE
Kang, Apakah Ini Wajar? Konsep Tag/Label di Mecha

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#223] [Q&A]
Bagaimana yah, cara menjelaskannya...

berikut ada halaman yang ane jadikan sebagai induk halamannya:
contohnya:
`https://www.lokersumatera.my.id/loker`

di dalam halaman diatas ane buat berbagai macam sub halamannya sesuai niche website ane om.
contohnya:
`https://www.lokersumatera.my.id/loker/pekanbaru`

kalau ane sertakan tag nya url nya akan seperti ini:
`https://lokersumatera.my.id/loker/tag/pekanbaru/1`

terus di dalam sub halaman `pekanbaru` ini isi beberapa postingan/ artikel.

Contohnya:
`https://lokersumatera.my.id/loker/pekanbaru/apotek-dr-misbah-rmc`

ane sertakan tag di di dalam halaman postingan/artikel diatas:

url tagnya jadi seperti:
`https://lokersumatera.my.id/loker/pekanbaru/tag/pekanbaru/1`

ane pikir untuk halaman ini `https://lokersumatera.my.id/loker/pekanbaru/apotek-dr-misbah-rmc` , kalau ane tambahin tag pekanbaru maka url nya setidaknya jadi url begini `https://lokersumatera.my.id/loker/tag/pekanbaru/1` atau lebih diatas parentnya `loker` bisa jadi seperti ini `https://lokersumatera.my.id/tag/pekanbaru/1`. apakah ini memang begini konsep tag di mecha kang? karena tag yang tampil tidak global. karena hal ini ane belum pede buat nampilin tag di mecha

maaf ya  kang, ane banyak tanya. mohon penjelasannya.

**Comment by taufik-nurrohman** [comment]
Tagging di Mecha menyesuaikan lokasi folder mas, tujuannya supaya fitur ini bisa diberlakukan ke semua listing halaman, di semua folder.

**Comment by taufik-nurrohman** [comment]
https://github.com/mecha-cms/x.tag/issues/5

Rencana:

~~~ txt
https://lokersumatera.my.id/tag/pekanbaru/1
~~~
